sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/E2E.Tests/Azure.Monitor.OpenTelemetry.Exporter.E2E.Tests.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.ApplicationInsights.Query" />
```

Comment:
I think this can be removed.

```suggestion
```

Reply:
Not right now. will do delete separately (I have not removed the test based on this package).

Reply:
@vishweshbankwar The linked issue isn't truly fixed until this package reference has been removed. I suggest changing your "fixes" comment to "addresses".

Reply:
Done - Thanks!